test/test_modules.py

```diff
@@ -35,6 +38,76 @@ def test_forward(self, device, dtype, module_info):
                     ref_outputs = reference_fn(m, *args, **kwargs)
                     self.assertEqual(outputs, ref_outputs)
 
+    # Tests passing factory kwargs (e.g. device / dtype) during module instantiation.
+    # They should be applied to any created parameters and buffers.
+    @modules(module_db)
+    def test_factory_kwargs(self, device, dtype, module_info):
+        module_cls = module_info.module_cls
+        module_inputs = module_info.module_inputs_func(module_info, device=device, dtype=dtype,
+                                                       requires_grad=False)
+        for module_input in module_inputs:
```

Comment:
Style nit: you might be able to unpack `module_inputs` in this for loop instead of on the line below, but whatever